packages/bot-web-ui/src/components/dashboard/bot-builder/bot-builder.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import RootStore from 'Stores/index';
+import AppStore from 'Stores/app-store';
+import { connect } from 'Stores/connect';
+import WorkspaceWrapper from './workspace-wrapper';
+
+interface BotBuilderProps {
```

Comment:
I suggest to follow the documentation on best practices of TS files to make sure a standard setup all over the repo.
https://wikijs.deriv.cloud/en/Frontend/deriv-app/Guides/typescript-migration

Reply:
done 🙇